src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -201,13 +201,53 @@
 					AccessKey="B"
 					Command="{x:Bind ViewModel.EmptyRecycleBinCommand, Mode=OneWay}"
 					IsEnabled="{x:Bind ViewModel.CanEmptyRecycleBin, Mode=OneWay}"
+					Visibility="{x:Bind ViewModel.CanEmptyRecycleBin, Mode=OneWay}"
 					Label="{helpers:ResourceString Name=EmptyRecycleBin}"
 					ToolTipService.ToolTip="{helpers:ResourceString Name=EmptyRecycleBin}">
 					<AppBarButton.Icon>
 						<FontIcon FontFamily="{StaticResource RecycleBinIcons}" Glyph="&#xEF88;" />
 					</AppBarButton.Icon>
 				</AppBarButton>
-				<AppBarButton
+                <AppBarButton
```

Comment:
Is the spacing off? Should use tab spaces.

Reply:
Done.